pom.xml

```diff
@@ -1056,6 +1094,81 @@
     </profile>
 
     <profile>
+      <id>jacoco</id>
```

Comment:
addition of this profile is not validated by Travis for this PR, while IMO would be really nice, especially that ``check`` goal currently fails